### PR TITLE
[HB-6381] Check integrated Mediation SDK version

### PR DESCRIPTION
**Pull request by chauduyphanvu**
The 1st of 2 PRs, the latter of which will remove the testing flags that are in place. After this first PR has been merged, I'll test to make sure it works.

This checks for the Mediation SDK version integrated with the demo app and notify Slack if the version is outdated.

### CODE REVIEW

**Comment by chauduyphanvu** [comment]
Here's how it'll look: https://chartboost.slack.com/archives/C03F39XQX34/p1697723452741869

File: .github/workflows/sdk-update-alert.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: Check Chartboost Mediation SDK Version
+
+on:
+  schedule:
+    # Runs every day at midnight
+    - cron: '0 0 * * *'
```
Comment (bichenwang) [inline .github/workflows/sdk-update-alert.yml:6]: just confirming, this will run at 5 pm pacific I believe

**Review by bichenwang — APPROVED** [review]
seems fine for now

File: .github/workflows/sdk-update-alert.yml (hunk 1 above)
Comment (chauduyphanvu) [reply .github/workflows/sdk-update-alert.yml:6]: yeah I mean it's midnight somewhere (UTC?). they need to support timezones. bump this: https://github.com/orgs/community/discussions/13454 😛

**Pull request merged**